Documentation/ColumnsConfig/Type/Input/Index.rst

```diff
@@ -25,6 +25,16 @@ appropriate length.
     *   :php:`renderType=inputLink`: Use the TCA type :ref:`link <columns-link>`
         instead.
 
+..  versionchanged:: 13.2
+    Tables with TCA columns set to `type="input"` do not
+    need an :file:`ext_tables.sql` entry anymore. The Core now
+    creates this field automatically. For short input fields allowing less
```

Comment:
Using the word "field" for a field within the TCA form is OK for me, but if we are talking about SQL and table columns I would prefer using "column". The second "field" in that row is OK for me as it relates to the TCE form.

```suggestion
    creates this column automatically. For short input fields allowing less
```